content/news/2024-09-13_haiku_r1_beta5.md

```diff
@@ -0,0 +1,8 @@
++++
+type = "news"
+title = "Haiku R1/beta5 has been released!"
+date = "2022-09-13 12:00:00Z"
+tags = ["release"]
++++
+
+After about a year and a half since the last beta, Haiku R1/beta5 has been released. See "**[Release Notes](/get-haiku/r1beta5/release-notes/)**" for the release notes, "[Press contact](/get-haiku/r1beta4/release-notes/#press-contact)", for press inquiries ... and "**[Get Haiku!](/get-haiku/r1beta5/)**" to skip all that and just download the release (or upgrade to it from an existing install!)
```

Comment:
```suggestion
After about a year and a half since the last beta, Haiku R1/beta5 has been released. See "**[Release Notes](/get-haiku/r1beta5/release-notes/)**" for the release notes, "[Press contact](/get-haiku/r1beta5/release-notes/#press-contact)", for press inquiries ... and "**[Get Haiku!](/get-haiku/r1beta5/)**" to skip all that and just download the release (or upgrade to it from an existing install!)
```

Reply:
done